ecs/utils.py

```diff
@@ -7,19 +8,40 @@
 logger = logging.getLogger(__name__)
 
 
-def h1(x): print("\033[1m\033[4m\033[94m{x}\033[0m\n".format(x=x))
+class Printer:
+    def __init__(self, *, prefix: str = "") -> None:
+        self.prefix = prefix
 
+    def h1(self, text: str) -> None:
+        print(f"\033[1m\033[4m\033[94m{self.prefix}{text}\033[0m\n")
 
-def h2(x): print("\033[1m\033[4m{x}\033[0m\n".format(x=x))
+    def h2(self, text: str) -> None:
+        print(f"\033[1m\033[4m{self.prefix}{text}\033[0m\n")
 
+    def success(self, text: str) -> None:
+        print(f"\033[92m* {self.prefix}{text}\033[0m\n")
 
-def success(x): print("\033[92m* {x}\033[0m\n".format(x=x))
+    def error(self, text: str) -> None:
+        print(f"\033[91mx {self.prefix}{text}\033[0m\n")
 
+    def info(self, text: str) -> None:
+        print(f"  {self.prefix}{text}\n")
 
-def error(x): print("\033[91mx {x}\033[0m\n".format(x=x))
 
+_default_printer = Printer()
+h1 = _default_printer.h1
+h2 = _default_printer.h2
+success = _default_printer.success
+error = _default_printer.error
+info = _default_printer.info
 
-def info(x): print("  {x}\n".format(x=x))
+
+def setup_dryrun(*, printer: t.Optional[Printer] = None) -> None:
```

Comment:
This is not best, but deploy-script is running one-time, so activating this effect globally is easier changes.